lib/node_modules/@stdlib/stats/base/dists/laplace/ctor/benchmark/benchmark.js

```diff
@@ -134,18 +147,25 @@ bench( pkg+'::get:b', function benchmark( bm ) {
 
 bench( pkg+'::set:b', function benchmark( bm ) {
 	var dist;
+	var len;
 	var mu;
 	var b;
+	var x;
 	var y;
 	var i;
 
 	mu = 2.0;
 	b = 3.0;
+	len = 100;
+	x = new Float64Array( len );
 	dist = new Laplace( mu, b );
+	for ( i = 0; i < len; i++ ) {
+		x[ i ] = uniform( EPS, 100.0 );
+	}
 
 	bm.tic();
 	for ( i = 0; i < bm.iterations; i++ ) {
-		y = ( 100.0*randu() ) + EPS;
+		y = x[ i % len ];
 		dist.b = y;
```

Comment:
```suggestion
		dist.b = y[ i % len ];
```

You do not need to declare a new variable `x` here. Accordingly, the above number generation should be modified.